static/js/map/map.routes.js

```diff
@@ -0,0 +1,265 @@
+                imageUrl = route.end_poi_image_url.replace(/^http:\/\//i, '//')
+                imageClass = 'pokestop-image'
+                imageOnclick = `onclick='showImageModal("${imageUrl}", "${route.name.replace(/"/g, '\\&quot;').replace(/'/g, '\\&#39;')}")'`
+            } // else keep default route_icon image
+        }
+        iconUrl = `static/images/routes/route_${marker.start ? 'start' : 'end'}.png`
+        routeTitle = `Route ${marker.start ? 'start' : 'end'}`
+    } else {
+        if (route.image) {
+            imageUrl = route.image.replace(/^http:\/\//i, '//')
+            imageClass = 'pokestop-image'
+            imageOnclick = `onclick='showImageModal("${imageUrl}", "${route.name.replace(/"/g, '\\&quot;').replace(/'/g, '\\&#39;')}")'`
+        } // else keep default route_icon image
+    }
+    const rDS = parseInt(route.route_duration_seconds,10)
+    const duration = `${Math.floor(rDS / 3600)}h ${lpad( Math.floor((rDS % 3600)/60), 2, '0')}m ${lpad( Math.floor(rDS % 60), 2, '0')}s`
+
+    const color = getRouteColor(route)
+    const routeDisplay = `
+        <hr style="height:5px;border-width:0;color:${color};background-color:${color}">
+        <div class='pokestop-container'>
+          <div class='pokestop-container-left'>
+            <div>
+              <img class='quest-image' src="${iconUrl}" width='64'/>
+            </div>
+          </div>
+          <div class='pokestop-container-right'>
+            <div class='title'>
+              ${i18n(routeTitle)}
+            </div>
+            <div class='info-container'>
+              <div>
+                ${i18n('Distance')}: <strong>${route.route_distance_meters} ${i18n('meters')}</strong>
+              </div>
+              <div>
+                ${i18n('Duration')}: <strong>${duration}</strong>
+              </div>
+              <div>
+                ${i18n('Reversible')}: <strong>${route.reversible}</strong>
+              </div>
+            </div>
+          </div>
+        </div>`
+
+    return `
+        <div>
+          <div class='pokestop-container'>
+            <div class='pokestop-container-left'>
+              <div>
+                <img class='${imageClass}' src='${imageUrl}' ${imageOnclick} width='64' height='64'>
+              </div>
+            </div>
+            <div class='pokestop-container-right'>
+              <div class='title'>
+                ${route.name}
+              </div>
+              <div>
+                ${i18n('Last scanned')}: <strong>${timestampToDateTime(route.last_updated)}</strong>
+              </div>
+              <div>
+                <a href='javascript:void(0);' onclick='javascript:openMapDirections(${route.start_poi_latitude},${route.start_poi_longitude},"${settings.mapServiceProvider}");' title='${i18n('Open in')} ${mapServiceProviderNames[settings.mapServiceProvider]}'><i class="fas fa-map-marked-alt"></i> ${route.start_poi_latitude.toFixed(5)}, ${route.start_poi_longitude.toFixed(5)}</a>
```

Comment:
All tooltips have the start lat long values. This should be adjusted for the 3 different tooltips.